modules/gapi/test/cpu/gapi_render_tests_cpu.cpp

```diff
@@ -0,0 +1,63 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2018 Intel Corporation
+
+
+#include "../test_precomp.hpp"
+#include "../common/gapi_render_tests.hpp"
+
+namespace opencv_test
+{
+
+INSTANTIATE_TEST_CASE_P(RenderTextTestCPU, RenderTextTest,
+                        Combine(Values(cv::Size(1280, 720),
+                                       cv::Size(640, 480),
+                                       cv::Size(128, 128)),
+                                Values("text"),
```

Comment:
just curious what if the text is empty? or "\0"?

Reply:
A blank text will be printed, this is an interesting case, but it increases the number of tests twice, I don't think it's worth it. What do you think ?

Reply:
I guess it's fine not to add it explicitly, just wondering whether you tried such stuff out to ensure there are no segfauls, unexpected behaviors, etc.

Reply:
consider resolved